requirements/specifications/intents/user-interest.md

```diff
@@ -0,0 +1,299 @@
+that time **MUST** be ignored. The timeout value is stored in the
+device's configuration manifest.
+
+To be notified when a user expresses interest in the currently displayed
+content, an App **MUST** provide the
+`xrn:firebolt:capability:discovery:interest` capability by enabling the
+`Discovery.onRequestUserInterest` notification.
+
+If there is a valid entity to return, then the method registered by the
+App **MUST** return the currently displayed entity meta-data.
+
+If there is no valid entity to return, then the method **MUST** throw an
+exception.
+
+If the foreground App returns a valid `EntityDetails` before the timeout,
+then, the returned value **MUST** be used.
+
+If there is no app registered the an error **MUST** be returned.
+
+### 3.3. Upstream User Interest Intent
+In some cases, e.g. a voice assistant, some upstream component will inform
+the platform that the user is interested in whatever is currently being presented.
+
+To do this, the upstream system **MUST** send a `Interest` intent, which describes the type of and reason for the interest.
+
+```json
+{
+  "action": "interest",
+  "data": {
+    "type": "interest",
+    "reason": "playlist"
+  }
+}
+```
+
+When a Firebolt platform receives this intent, it **SHOULD** initiate the platform's [user interest flow](#4-user-interest-from-a-platform-ux).
+
+### 3.4. User Interest Bulk Updates
+
+Sending bulk interest updates, e.g. Entities the user expressed interest
+in on a different platform, is not supported.
+
+## 4. Core SDK APIs
+
+The following APIs are exposed by the Firebolt Core SDK as part of the
+`Discovery` module.
+
+### 4.1. InterestType
+This is an enum with the following values:
+
+- `"interest"`
+- `"disinterest"`
+
+### 4.2. InterestReason
+This is an enum with the following values:
+
+| Reason | Description |
+| ------ | ----------- |
+| `playlist` | Interested in adding to a list |
+| `reaction` | Interested in submitting a reaction, e.g. like or dislike |
+| `recording` | Interest in scheduling a recording |
+| `share` | Interest in sharing the content on social media |
+
+### 4.3. Discovery.userInterest
+
+This is a push API that allows Apps to push entities that the user has
+expressed interest in to the platform.
+
+To push an entity that the user is interested in pass an `EntityDetails`
+object to the method:
+
+```typescript
+Discovery.userInterest(type: InterestType, reason: InterestReason, entity: EntityDetails): Promise<void>
+```
+
+### 4.4. Discovery Interest Provider
+To respond to requests for the current entity, because the user has
+expressed interest in some way that the platform manages, register a
+provider:
+
+```typescript
+interface IDiscoveryInterestProvider {
+  function userInterest(type: InterestType, reason: InterestReason): Promise<EntityDetails>
+}
+
+Discovery.provide("xrn:firbolt:capability:discovery:interest", IDiscoveryInterestProvider)
+```
+
+### 4.5. InterestIntent
```

Comment:
Is this still needed? Don't think we are still modeling this as an intent anymore?

Reply:
it is needed, but I'll likely move it to the other doc